### PR TITLE
Donate page completed

### PR DESCRIPTION
**Pull request by dawad725**

### CODE REVIEW

File: src/pages/Donate.tsx [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                <div className={styles.container}>
+                    <br></br>
+                    <Typography
+                        variant="body1"
+                        component="p"
+                        align="center"
+                        color="secondary"
+                    >
+                        Any additional donations beyond deployment towards platform maintenance and
+                    </Typography>
```
Comment (ComeTiss) [inline src/pages/Donate.tsx]: create a small component before return() to reduce code duplication, like:

const TextBody1 =  (content: string) => (
                  <Typography  ....>
                       {content}
                    </Typography>
)

File: src/pages/Donate.tsx [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+                    <br></br>
+                </div>
+                <div className={styles.gridContainer}>
+                    <Grid container className={styles.root}>
+                        <Grid item xs={12} md={3}>
+                            <div className={styles.circle}>1</div>
+                            <Box className={styles.box}>
+                                Site hosting and server maintenance
+                            </Box>
+                        </Grid>
```
Comment (ComeTiss) [inline src/pages/Donate.tsx]: predfine in separate const, to avoid code duplication (see comment below for ex)

**Review by ComeTiss — COMMENTED** [review]
Just fix the code duplication and good to go :)

**Comment by ComeTiss** [comment]
you need to run **npm run lint:fix** to fix linter

File: src/pages/Donate.tsx [hunk 3]
```diff
@@ -97,25 +97,23 @@ const useStyles = makeStyles(theme => ({
...
-  //     transform: "translateX(-50%)",
-  //     bottom: "0"
-  // }
+  footerContainer: {
+    background: "rgba(255, 255, 255, 0.6)",
+    padding: "0rem",
+    //   display: "flex",
+    // minHeight: "60vh",
+    // marginBottom: "50vh",
+    // marginTop: "-180px",
```
Comment (ComeTiss) [inline src/pages/Donate.tsx:108]: remove if you dont use it

**Review by ComeTiss — APPROVED** [review]
Niceeee :) You just need to fix the linter & probably rebase and good to merge

File: src/pages/Donate.tsx (hunk 3 above)
Comment (dawad725) [reply src/pages/Donate.tsx:108]: planning on it, still fine tuning.

**Pull request merged**